src/lib/blueprint-features.ts

```diff
@@ -0,0 +1,109 @@
+import { Blueprint } from 'src/stores/wpcom-api';
+
+export interface UnsupportedFeature {
+	type: 'step' | 'property';
+	name: string;
+	reason: string;
+}
+
+/**
+ * List of blueprint features that are not supported in Studio
+ */
+export const UNSUPPORTED_BLUEPRINT_FEATURES: UnsupportedFeature[] = [
```

Comment:
We don't use it outside
```suggestion
const UNSUPPORTED_BLUEPRINT_FEATURES: UnsupportedFeature[] = [
```